examples/js/vr/WebVR.js

```diff
@@ -95,6 +95,12 @@ var WEBVR = {
 
 		}, false );
 
+		window.addEventListener( 'beforeunload', function ( event ) {
+
+			if (effect.isPresenting) effect.exitPresent();
+
+		});
```

Comment:
Yeah, that's no longer the case.   It will default to `False`   Was an old habit for me too :)